packages/cli/src/cmds/init/initUtils.ts

```diff
@@ -336,3 +345,29 @@ export async function prepareWorkingDirectory(): Promise<void> {
     }
   }
 }
+
+/**
+ * In case when TypeScript version is used, default source file
+ * will be updated if previous was not correct.
+ *
+ * @param yargsArgv - The Yargs arguments object.
+ * @returns Modified Yargs arguments object.
+ */
+export function correctDefaultArgs(yargsArgv: Arguments): Arguments {
+  if (yargsArgv.template === 'typescript' && yargsArgv.src === 'src/index.js') {
```

Comment:
Use the enum here!
`if (yargsArgv.template === TemplateType.TypeScript && (...)`

Reply:
Using it here and everywhere else.

Reply:
Updated in few more places in the last commit.